LibreNMS/OS/Ewc.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Ewc.php
+ *
+ * Extreme Wireless Controller
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2017 James Andrewartha
+ * @author     James Andrewartha <trs80@ucc.asn.au>
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessApCountDiscovery;
+#use LibreNMS\Interfaces\Discovery\Sensors\WirelessCcqDiscovery;
```

Comment:
Yeah, I was leaving them to remind me to implement them. I'll remove before my final commit.